assets/styles/layout/_options.scss

```diff
@@ -123,6 +154,25 @@
     text-transform: uppercase;
   }
 
+  .hide-on-mobile {
+    @include media-breakpoint-down(lg) {
+      display: none;
+    }
+  }
```

Comment:
This could be a global css thing right?

Reply:
Probably could be; is there a better place to put it? Or an existing thing like this that I could use instead?

Reply:
I put it in _breakpoints.scss with the other screen size stuff.